cibuildwheel/windows.py

```diff
@@ -146,9 +150,9 @@ def shell(args, env=None, cwd=None):
         if test_command:
             # set up a virtual environment to install and test from, to make sure
             # there are no dependencies that were pulled in at build time.
-            shell(['pip', 'install', 'virtualenv'], env=env)
+            shell(['pip', 'install', 'virtualenv'] + dependency_constraint_flags, env=env)
             venv_dir = tempfile.mkdtemp()
-            shell(['python', '-m', 'virtualenv', venv_dir], env=env)
+            shell(['python', '-m', 'virtualenv', '--no-download', venv_dir], env=env)
```

Comment:
in test suite I do not see any test if created virtualenv preserve package version.

Reply:
See my https://github.com/joerick/cibuildwheel/pull/256/files#r386025737 above - I'm not sure how easy that would be. I have added a comment about determinism here, though.